apps/website/components/ui/GlobalToast.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <v-snackbar
+    v-model="isVisible"
+    :timeout="toast?.timeout || 4000"
+    :color="toastColor"
+    :persistent="toast?.persistent"
+    location="top right"
+    elevation="6"
+    rounded="lg"
+    class="toast-snackbar"
+    multi-line
+  >
+    <div class="d-flex align-center">
+      <v-icon :icon="toastIcon" class="mr-3" size="20" />
+      <span class="toast-message">{{ toast?.message }}</span>
+    </div>
+
+    <template v-slot:actions>
+      <v-btn
+        aria-label="Close notification"
+        icon="mdi-close"
+        size="small"
+        variant="text"
+        @click="hideToast"
+      />
+    </template>
+  </v-snackbar>
+</template>
+
+<script setup lang="ts">
+  import { computed } from 'vue';
+  import { useToast } from '~/composables/useToast';
+
+  const { toast, hideToast } = useToast();
+
+  const isVisible = computed({
+    get: () => !!toast.value,
+    set: value => {
+      if (!value) hideToast();
+    }
+  });
+
+  const toastColor = computed(() => {
+    switch (toast.value?.type) {
+      case 'success':
+        return 'success';
+      case 'error':
+        return 'error';
+      case 'warning':
+        return 'warning';
+      case 'info':
+      default:
+        return 'primary';
+    }
+  });
+
+  const toastIcon = computed(() => {
+    switch (toast.value?.type) {
+      case 'success':
+        return 'mdi-check-circle';
+      case 'error':
+        return 'mdi-alert-circle';
+      case 'warning':
+        return 'mdi-alert';
+      case 'info':
+      default:
+        return 'mdi-information';
+    }
+  });
+</script>
+
+<style scoped>
+  .toast-snackbar {
+    z-index: 9999;
+  }
+
+  .toast-message {
+    font-weight: 500;
```

Comment:
We have some generic typography styles, so ideally it is better to avoid custom styles in the components. In case they are really required, it is important to use variables from apps/website/assets/styles/_variables.scss